addOns/fuzz/src/main/javahelp/org/zaproxy/zap/extension/fuzz/resources/help/contents/api.html

```diff
@@ -0,0 +1,173 @@
+    <a href="https://github.com/zaproxy/zap-api-docs/fuzz">https://github.com/zaproxy/zap-api-docs/fuzz</a>
+    <br>
+</p>
+<h3>Message Processors</h3>
+<p>
+    The default processor enabled by default for all requests is “Request Content-Length Updater”
+    which updates the content-length field according to the attack payload. Option to add more
+    processors will be added in future releases.
+</p>
+<h3>Fuzzer Options</h3>
+<p>
+    Currently these fuzzer options can be changed through the FuzzAPI but in future releases
+    more options will be added.
+</p>
+<ul>
+    <li>Retries on IO error</li>
+    <li>Max. error allowed</li>
+    <li>Payload Replacement Strategy</li>
+    <li>Concurrent scanning threads per scan</li>
+    <li>Delay when fuzzing (in milliseconds)</li>
+</ul>
+Note: options changed after the fuzzer has been initialised will remain same for that fuzzer.
+<h3>Simple Http Fuzzer</h3>
+<p>The simpleHttpFuzzer fuzzes one fuzz location, with inputs</p>
+<table style="width:100%" border="1px solid black" border-collapse="collapse">
+    <tr>
+        <th>Parameter</th>
+        <th>Type</th>
+        <th>Example</th>
+    </tr>
+    <tr>
+        <td>messageId </td>
+        <td>int</td>
+        <td>7</td>
+    </tr>
+    <tr>
+        <td>requestLocation</td>
+        <td>string
+        </td>
+        <td>“body” or “header”</td>
+    </tr>
+    <tr>
+        <td>fuzzLocation</td>
+        <td>string (“int start: int end”)</td>
+        <td>8:15 or 12:18</td>
+    </tr>
+    <tr>
+        <td>payloadPath</td>
+        <td>string</td>
+        <td>“/home/users/payloads.txt”</td>
+    </tr>
+</table>
+<br>
+The parameter fuzzLocation is the start and end number of the character which is replaced
+by the payload.<br>
+For example:
+<p style="font-family:'Courier 10 Pitch',cursive">POST /cgi-bin/process.cgi HTTP/1.1
+    <br>User-Agent: Mozilla/4.0 (compatible; MSIE5.01; Windows NT)
+    <br>Host: www.example.com
+    <br>Content-Type: application/x-www-form-urlencoded
+    <br>Content-Length: length
+    <br>Accept-Language: en-us
+    <br>Accept-Encoding: gzip, deflate
+    <br>Connection: Keep-Alive
+    <br>licenseID=string&amp;content=string&amp;/paramsXML=string
+</p>
+<p>In this HTTP POST request, the top and bottom are header and body respectively.
+    <br><br>To fuzz the value of licenseID in the body the value of string needs to be replaced.
+</p>
+    <img src="images/simple_http_fuzzer_example.png" alt="Simple Http Fuzzer Example" width="300">
+<p>
+    <br><br>The messageId is the History Id of the message, requestLocation is body and fuzzLocation is
+    what characters should be replaced, payloadPath is the path of the file containing the payloads.
+</p>
+<h3>Multiple Payload Fuzzer</h3>
+<p>The multiple payload fuzzer as per name fuzzes multiple fuzz Locations with multiple payload
+    types. It takes input as the message Id and a certain Json file with a schematic to make the
```

Comment:
JSON in caps?

(Probably applies elsewhere as well....in written form I believe acronyms should use full caps)